src/global.scss

```diff
@@ -0,0 +1,60 @@
+@mixin grid3($cell1,$cell2,$cell3,$gap) {
+display: grid;
+grid-template-columns: $cell1 $cell2 $cell3;
+column-gap: $gap;
+justify-content: left;
+align-content: flex-start;
+}
+@mixin resetMarginPadding {
+margin: 0;
+padding: 0;
+}
+@mixin inputs($color,$top,$bot){
+border-radius: 3px;
+border: $borderColor;
+height: 40px;
+width: 80%;
+justify-content: center;
+align-items: center;
+background-color: $color;
+margin-top: $top;
+margin-bottom: $bot;
+}
+@mixin image ($ejeZ){
+position: absolute;
+z-index: $ejeZ;
+width: 50%;
+}
+@mixin backimag ($ejeX,$ejeY,$bordeR,$url){
+height: $ejeX;
+width: $ejeY;
+border-radius: $bordeR;
+margin: 5px;
+background-image: url($url);
+background-size: cover;
+background-position: center;
+}
+@mixin flex ($direction,$xPos,$ypos,$color,$borde,$anchor){
+display: flex;
+flex-direction: $direction;
+justify-content: $xPos;
+align-items: $ypos;
+width: $anchor;
+background-color: $color;
+border: #dbdbdb $borde solid;
+}
+@mixin aHref ($color,$fonts){
+text-decoration: none;
+color: $color;
+font-weight: 700;
+font-family: $fonts;
+justify-self: center;
+align-self: center;
+}
+
+@mixin list ($color,$fonts,$dist){
+    list-style: none;
+    color: $color;
+    font-family: $fonts;
+    margin-bottom: $dist;
+}  
```

Comment:
Como recomendación, siempre se debe dejar un salto de linea en los ficheros. Si usamos una herramienta como Prettier esto se hará de forma automática siempre.

https://stackoverflow.com/questions/2287967/why-is-it-recommended-to-have-empty-line-in-the-end-of-a-source-file#:~:text=The%20empty%20line%20in%20the,can%20handle%20the%20EOF%20marker.